src/Console/ConsoleServiceProvider.php

```diff
@@ -65,5 +66,9 @@ public function boot(Container $container)
             $event = $schedule->command($scheduled['command'], $scheduled['args']);
             $scheduled['callback']($event);
         }
+
+        $this->container->make('flarum.locales')->getTranslator()->getCatalogue(
+            $this->container->make(SettingsRepositoryInterface::class)->get('default_locale', 'en')
```

Comment:
Should use $container, not $this->container. Also, why do we need to call `getCatalogue`?

Reply:
the `getCatalogue` method is needed for the catalogue to be written to cache/disk

Reply:
```suggestion
            $container->make(SettingsRepositoryInterface::class)->get('default_locale', 'en')
```